src/test/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExternalApiExceptionTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.professionalapi.controller.advice;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+
+import org.junit.Test;
+
+public class ExternalApiExceptionTest {
+
```

Comment:
what is the difference between ExternalApiExceptionTest.java and GetUserProfileExceptionTest  classes?

Reply:
They test different exception classes. While the test code is the same, because these exceptions have their own classes, we require separate test classes as per the QA Concerns where each class should have its own corresponding test class. I can, however, easily put them into the same test class if you think that is preferable.

Reply:
After discussing with @shashank-rastogi I have addressed this in the following commit: https://github.com/hmcts/rd-professional-api/pull/450/commits/bb8cc54db87e6e7a5e9310b945532bee7c100802